packages/clay-data-provider/src/__tests__/index.tsx

```diff
@@ -0,0 +1,316 @@
+/*global fetchMock*/
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import DataProvider from '../';
+import LRU from 'lru-cache';
+import React from 'react';
+import {cleanup, render, wait} from 'react-testing-library';
+import {FetchMock} from 'jest-fetch-mock';
```

Comment:
Is this even needed? I noticed it wasn't called anywhere and the fetchMock global is already instantiated in `setupTests.ts`

Reply:
Yeah, this is actually a hack, for some reason typescript is not recognizing the typos of `setupTests` and was crashing at compile time due to` fetch` mocks.